bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx

```diff
@@ -88,6 +90,7 @@ class ModalFullscreen extends Component {
         </div>
         <div id="modalDismissDescription" hidden>{dismiss.description}</div>
         <div id="modalConfirmDescription" hidden>{confirm.description}</div>
+        <ToastContainer />
```

Comment:
This is the second `ToastContainer`, and they both are triggered.

![notify](https://user-images.githubusercontent.com/5004762/49827418-44059a00-fd57-11e8-8a78-57b94592f926.png)

Reply:
But the first one is covered by the modal window. I cannot find a good place to put that ToastContainer. 
Add another ToastContainer just a dirty workaround.

Reply:
I found a solution. Changes in #5918 cause toasts cannot show in mobile browser and Chrome. It'd better to set body position to fixed and change back #main to relative.